mono/mini/tramp-amd64.c

```diff
@@ -1002,11 +1014,20 @@ mono_arch_get_interp_to_native_trampoline (MonoTrampInfo **info)
 	for (i = 0; i < FLOAT_RETURN_REGS; i++)
 		amd64_sse_movsd_membase_reg (code, AMD64_R11, MONO_STRUCT_OFFSET (CallContext, fregs) + i * sizeof (double), i);
 
-	amd64_mov_reg_reg (code, AMD64_RSP, AMD64_RBP, 8);
+#if TARGET_WIN32
+	amd64_lea_membase (code, AMD64_RSP, AMD64_RBP, 0);
```

Comment:
I don't understand why this needs to change, given that the prologue appears to stay the same.

Reply:
Win x64 ABI only allows a couple of instructions in the epilog when using a frame pointer to correctly work with OS unwinder.

Reply:
@lateralusX is correct.
Epilogues must be detected via runtime disassembly and the disassembler is exceedingly minimal.

https://docs.microsoft.com/en-us/cpp/build/prolog-and-epilog?view=vs-2019